lib/src/simple_chips_input.dart

```diff
@@ -101,11 +104,14 @@ class SimpleChipsInput extends StatefulWidget {
   /// Callback when a chip is deleted. Returns the deleted chip content and index.
   final void Function(String, int)? onChipDeleted;
 
+  final void Function(String)? onChipAdded;
```

Comment:
Can you please add comments here, so it's easy for the devs to understand?
Adding a comment here also adds it to the API reference in pub dev